docs/user/pings/custom.md

```diff
@@ -220,6 +220,16 @@ Pings.search.Submit(
 
 </div>
 
+<div data-lang="Rust" class="tab">
+
+```Rust
+use glean::Pings;
+
+Pings::search.submit(Pings::searchReasonCodes::performed);
```

Comment:
```suggestion
pings::search.submit(pings::SearchReasonCodes::Performed);
```
we went for lowercase module and ping names, but the reason codes will be an enum, so they should be CamelCased.